AABB_tree/include/CGAL/AABB_tree.h

```diff
@@ -829,14 +830,26 @@ namespace CGAL {
   template<typename Tr>
   bool AABB_tree<Tr>::build_kd_tree()
   {
+    Bbox_3 bb;
+    for(const Primitive& p : m_primitives){
+      bb = bb + Helper::get_reference_point(p, m_traits).bbox();
+    }
+    double dmin = (std::min)((std::min)(bb.xmax()-bb.xmin(), bb.ymax()-bb.ymin()),bb.zmax()-bb.zmin());
+    dmin /= 30.0;
```

Comment:
This gives `dmin==0` if the point set is flat in one of the three axis, and then:
```
terminate called after throwing an instance of 'CGAL::Precondition_exception'
  what():  CGAL ERROR: precondition violation!
Expr: epsilon > 0
File: [...]]/cgal/Point_set_processing_3/include/CGAL/grid_simplify_point_set.h
Line: 228
```